startly_style.css

```diff
@@ -0,0 +1,218 @@
+@import url('http://fonts.googleapis.com/css?family=Economica');
+@import url('http://fonts.googleapis.com/css?family=Monoton');
+@import url('http://fonts.googleapis.com/css?family=Tangerine');
+
+
+* {
+  font-family: Economica;
+  background-color: seashell;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+}
+
+.topper {
```

Comment:
For the nav bar, since it's a 1D element, another way to go is to use Flexbox, and set margin-auto: left for the second list element to get both "Try Now!" and "About" to move to the far right.